### PR TITLE
Hotfix/layouts review

### PR DESCRIPTION
**Pull request by catherine-jones**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,108 +1,101 @@
...
-  <%= javascript_include_tag "https://code.jquery.com/jquery-1.4.min.js" %>
-  <!--<link rel="stylesheet" href="/assets/css/ie8.css">-->
-  <%= stylesheet_link_tag 'ie8', media: 'all', 'data-turbolinks-track': 'reload' %>
-  <!--<script src="/third-party/selectivizr.js"></script>-->
-  <!--<script src="https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.js"></script>-->
-  <%= javascript_include_tag 'selectivizr', "https://cdnjs.cloudflare.com/ajax/libs/respond.js/1.4.2/respond.js" %>
-  <![endif]-->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>undefined</title> <!--< TODO this shoudld be dynamical generated using Ruby-->
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: can we set this statically as something generic for now like "Australian Government Performance Dashboards"? 
Great "TODO" comment 👍

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,108 +1,101 @@
...
+    <meta name="description" content=""> <!--< TODO this shoudld be dynamical generated using Ruby-->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <%= stylesheet_link_tag "https://maxcdn.bootstrapcdn.com/font-awesome/4.6.0/css/font-awesome.min.css" %>
+    <link rel="stylesheet" href="vendor/assets/css/index.css">
+    <script src="/third-party/modernizr.min.js"></script>
+    <!--[if IE 8]>
+    <%= javascript_include_tag "https://code.jquery.com/jquery-1.4.min.js" %>
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: can we use cdnjs jquery1.4?

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,108 +1,101 @@
...
-  <script>window.ga = window.ga || function () {
-        (ga.q = ga.q || []).push(arguments)
-      };
-  ga.l = +new Date;
-  ga('create', 'UA-61222473-3', 'auto');
-  // Autotrack plugins
-  ga('require', 'eventTracker');
-  ga('require', 'outboundLinkTracker');
+    <!-- Google Analytics -->
+    <script>dataLayer = [
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: add TODO - this should be dynamic

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,108 +1,101 @@
...
-        <!--[if IE 8]><!--<img src="/assets/gov-performance.png" alt="gov.au" height="40">--><%= image_tag "gov-performance.svg", alt: "gov.au", height: "40" %><![endif]-->
-      </a>
-      <div class="toggle-group">
-        <span class="hidden-on-small">High contrast mode</span><input id="switch-mode" type="checkbox" name="switch-mode"
-                                                                      tabindex="0"><label for="switch-mode"><span class="aural"></span></label>
-        <div class="onoffswitch pull-right" aria-hidden="true">
-          <div class="onoffswitch-label">
-            <div class="onoffswitch-inner"></div>
-            <div class="onoffswitch-switch"></div>
+  <body class="#{bodyClass}">
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: add TODO - this needs to be dynamic

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,108 +1,101 @@
...
+      <div class="skiptocontent"><a href="#main">Skip to main content</a></div>
+      <div class="prototype-warning">
+        <div class="prototype-warning-text">This is a working alpha and has limited functionality. Service data shown is
+          accurate as at the time it was published.
+        </div>
+      </div>
+      <div class="logo-primary">
+        <div class="container">
+          <a class="site-title block" href="/">
+            <!--<img class="no-ie8" src="/assets/gov-performance.svg" alt="gov.au">-->
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: can we remove this?

File: app/views/layouts/application.html.erb [hunk 6]
```diff
@@ -1,108 +1,101 @@
...
+            <%= image_tag "gov-performance.svg", class: "no-ie8", alt: "gov.au"%>
+            <!--[if IE 8]>
+            <%= image_tag "gov-performance.svg", alt: "gov.au", height: "40" %>
+            <![endif]-->
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: image tag should be png if ie8

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,108 +1,101 @@
...
-<!--[if !IE 8]><!-->
-<!--<script src="/assets/javascripts/index.js"></script>-->
-<%= javascript_include_tag 'index' %>
-<![endif]-->
+    <footer>
+      <section>
+        <!--[if IE 8]>
+        <%= image_tag "coatofarms.png",  style:"height: 76px",  alt: "Commonwealth Coat of Arms" %>
+        <![endif]-->
+        <%= image_tag "coatofarms.svg", style: "height: 76px", alt: "Commonwealth Coat of Arms" %>
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: need to wrap this is if not ie8

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,108 +1,101 @@
...
+          <li><a href="https://www.dto.gov.au/standard/11-measure-performance/">About Performance Measurement</a></li>
+          <li><a href="/copyright">Copyright</a></li>
+          <li><a href="https://www.dto.gov.au/blog/">DTO Blog</a></li>
+          <li><a href="https://github.com/AusDTO/d3-charts-dto">Our Code</a></li>
+          <li><a href="/feedback">Please give feedback</a></li>
+        </ul>
+      </section>
+    </footer>
+    <!--[if !IE 8]>
+    <%= javascript_include_tag 'index' %>
```
Comment (elisechant) [inline app/views/layouts/application.html.erb]: does this need a path to asset or it's implied?

**Comment by elisechant** [comment]
hey Cath, looking a lot tidier 👍 A couple of quick things but mostly LGTM. 
We also need to sync back on the productotype because some of these tags have changed slightly since.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (catherine-jones) [inline app/views/layouts/application.html.erb]: yeah for sure

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (catherine-jones) [inline app/views/layouts/application.html.erb]: @elisechant are you referring to the IE conditional for outdated browser?

File: app/views/layouts/application.html.erb (hunk 6 above)
Comment (catherine-jones) [inline app/views/layouts/application.html.erb]: @elisechant good catch thankd

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (tobyhede) [inline app/views/layouts/application.html.erb]: Can make it dynamic in the template now:

```
<%= content_for(:title) || 'Sensible Default %>
```

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (tobyhede) [inline app/views/layouts/application.html.erb]: Do you mean come from an ENV var or config? 
That could be done now.

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (elisechant) [inline app/views/layouts/application.html.erb]: @catherine-jones no the commented out image

File: app/views/layouts/application.html.erb (hunk 3 above)
Comment (elisechant) [inline app/views/layouts/application.html.erb]: is there an application controller?

**Comment by elisechant** [comment]
LGTM :)

**Pull request merged**